packages/languages/src/browser/window-impl.ts

```diff
@@ -56,8 +56,17 @@ export class WindowImpl implements Window {
         return {
             append: outputChannel.append.bind(outputChannel),
             appendLine: outputChannel.appendLine.bind(outputChannel),
-            show: function (preserveFocus?: boolean): void {
-                // no-op
+            show: async (preserveFocus?: boolean) => {
```

Comment:
thanks! very cool!

one thing: for Java files it doesn't select the source correctly, stays on TypeScript.

Reply:
e.g. create and open an empty java file. JDT LS will throw, you should see it there.